app/geant-exporter/BremsstrahlungProcess.hh

```diff
@@ -0,0 +1,68 @@
+//----------------------------------*-C++-*----------------------------------//
+// Copyright 2021 UT-Battelle, LLC, and other Celeritas developers.
+// See the top-level COPYRIGHT file for details.
+// SPDX-License-Identifier: (Apache-2.0 OR MIT)
+//---------------------------------------------------------------------------//
+//! \file BremsstrahlungProcess.hh
+//! \brief Bremsstrahlung process class.
+//---------------------------------------------------------------------------//
+#pragma once
+
+#include <string>
+#include <G4VEnergyLossProcess.hh>
+
+namespace geant_exporter
+{
+//---------------------------------------------------------------------------//
+/*!
+ * Bremsstrahlung model selection.
+ */
+enum class BremsModelSelection
+{
+    seltzer_berger,
+    relativistic,
+    all
+};
+
+//---------------------------------------------------------------------------//
+/*!
+ * Electron/positron Bremsstrahlung process class derived from
+ * \c G4eBremsstrahlung . The need for a new process class is to add the option
+ * to manually select individual models.
+ */
+class BremsstrahlungProcess : public G4VEnergyLossProcess
+{
+  public:
+    // Construct with model selection
+    BremsstrahlungProcess(BremsModelSelection selection,
+                          const std::string&  name = "eBrem");
```

Comment:
I don't think we're ever going to adjust the name of this process... I'd just pass the hard-coded name to the parent constructor.

Either way you also need to mark this constructor as explicit.